string Needs
string object
string SelfMood
string ObjMood
string Speech